projects/plugins/jetpack/extensions/blocks/videopress/v6/components/videopress-uploader/index.js

```diff
@@ -220,6 +220,36 @@ const VideoPressUploader = ( { attributes, setAttributes } ) => {
 		startUpload( file );
 	}
 
+	const getErrorMessage = () => {
+		if ( ! uploadErrorData ) {
+			return '';
+		}
+
+		let errorMessage = __(
+			'An error was encountered during the upload. Check your network connection.',
+			'jetpack'
+		);
+
+		// Let's give this error a better message.
+		if ( uploadErrorData?.data?.message === 'Invalid Mime' ) {
```

Comment:
Is it the safest way to detect the MIME error? just wondering...

Reply:
yes.. we could also change the error message in the server side, but I'm always unsure if someone else is expecting it and whether we could break anything if we changed the response

Reply:
I was [taking a look at the current implementation](https://github.com/Automattic/jetpack/blob/fbeb5fdc370c0ba84adcee45e683442ab6e5ba45/projects/plugins/jetpack/extensions/blocks/videopress/resumable-upload/index.js#L115-L120), just in case.